pkg/virtual/generic/rootapiserver/root_apiserver.go

```diff
@@ -295,8 +301,8 @@ func NewRootAPIConfig(kubeClientConfig *rest.Config, rootAPIServerBuilder RootAP
 		GenericConfig: genericConfig,
 		ExtraConfig: RootAPIExtraConfig{
 			InformerStart:                      func(stopCh <-chan struct{}) {
-				for _, inf := range rootAPIServerBuilder.AdditionalInformers {
-					inf.Start(stopCh)
+				for _, informerStart := range rootAPIServerBuilder.InformerStarts {
+					informerStart(stopCh)
```

Comment:
This might warrant some explanation

Reply:
nit: s/informerStart/startInformer/, i.e. verbs